core/domain/exp_services_test.py

```diff
@@ -48,6 +48,15 @@
 
 
 def _count_at_least_editable_exploration_summaries(user_id):
+    """Counts exp summaries that are at least editable by the given user.
+
+    Args:
+        user_id: unicode. The id of the given user.
```

Comment:
I think the data type should be `str`. Are you sure here?

Reply:
@vibhor98 `type(user_id)`method returns `unicode`

Reply:
Ah... ok. Thanks for confirming! Can you make all other review comments ASAP? Else the PR looks fine!